### DISCUSSION TITLE
compact thread view?

### DISCUSSION THREAD

**Discussion by rafnov** [FossifyOrg/General-Discussion#497] [Q&A]
The list of messages in app can show details that I don't need and so they take space so much that I can see only 9 threads on one screen. To compare - QUIK hides those details (like message time) and shows 12 threads on 1 screen. How can it be done in Fossify messages app?

**Comment by Aga-C** [comment]
We don't have a compact view. You can only change the font size in settings to make thread list more compact, but it will also affect the rest of the app. There's also already a similar request: https://github.com/FossifyOrg/Messages/issues/110.

**Reply by rafnov** [reply]
Thx, Aga-C, for pointing correct issue thread.
